frontend/src/app/modules/lessons/lessons-page/lessons-page.component.ts

```diff
@@ -68,29 +78,32 @@ export class LessonsPageComponent implements OnInit, OnChanges {
     }
 
     getLessons() {
-        this.lessonService.getFilteredLessons({
-            languageLevels: Array.from(this.selectedLanguageFilters).map((level: string) => Object.values(LanguageLevels).indexOf(level)),
-            tags: Array.from(this.selectedTopicsFilters).map((topic) => ({ name: topic })),
-            date: new Date(this.selectedDateFilter.toISOString().slice(0, 10)),
-        }).subscribe((response: ILesson[]) => {
-            this.lessons = [];
-
-            response.forEach(lesson => {
-                this.lessons.push({
-                    id: lesson.id,
-                    imgPath: lesson.mediaPath,
-                    videoId: 'xqAriI87lFU',
-                    title: lesson.name,
-                    time: lesson.startAt.split('T')[1].substring(0, 5).replace(':', '.'),
-                    tutorAvatarPath: '../../../../assets/lesson-mocks/Photo )Patient).png',
-                    tutorFlagPath: '../../../../assets/lesson-icons/canada-test-flag.svg',
-                    tutorName: 'Roger Vaccaro',
-                    topics: lesson.tags.map((tag) => tag.name),
-                    viewersCount: lesson.subscribersCount,
-                    level: langLevelsSample[lesson.languageLevel].title,
-                    isDisabled: (new Date() > new Date(lesson.startAt)),
+        this.lessonService
+            .getFilteredLessons({
+                languageLevels: Array.from(this.selectedLanguageFilters).map((level: string) =>
+                    Object.values(LanguageLevels).indexOf(level)),
+                tags: Array.from(this.selectedTopicsFilters).map((topic) => ({ name: topic })),
```

Comment:
We create an array of numbers for the post request, and the index is an enam number. Vlad made the task about the lessons, and I made a change there so that my task about the button 'Close All' worked correctly